antora-ui-camel/src/css/doc.css

```diff
@@ -667,3 +667,36 @@ table.tableblock thead {
   background: var(--color-smoke-90);
   font-weight: var(--body-font-weight-bold);
 }
+
+/* Status badges used in Camel Quarkus extension pages */
+div.badges p {
+  text-align: right;
+}
+
+div.badges span {
+  color: var(--color-white);
+  padding: 2px 6px 2px 6px;
```

Comment:
Prefer using relative values, e.g. `rem`, test by changing the base font size to see if the proportions work. This is to help folk that configure their browsers with larger/smaller default text sizes. This works for almost any usage, though sometimes using `px` makes sense, like for borders.

Reply:
Thanks, done in d140a11